mne/io/base.py

```diff
@@ -850,9 +856,10 @@ def apply_function(self, fun, picks, dtype, n_jobs, *args, **kwargs):
                                       for p in picks)
             for pp, p in enumerate(picks):
                 self._data[p, :] = data_picks_new[pp]
```

Comment:
But the default is handled internally already, it tests for `None` and other types.